olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -9,3 +9,17 @@ entries:
   - name: "serverless-operator.v1.29.0"
     replaces: "serverless-operator.v1.28.0"
     skipRange: ">=1.28.0 <1.29.0"
+---
+schema: olm.channel
+name: serverless-1.29
+package: serverless-operator
+entries:
+  - name: "serverless-operator.v1.27.0"
+  - name: "serverless-operator.v1.28.0"
+    replaces: "serverless-operator.v1.27.0"
+    skipRange: ">=1.27.0 <1.28.0"
+  - name: "serverless-operator.v1.29.0"
+    replaces: "serverless-operator.v1.28.0"
+    skipRange: ">=1.28.0 <1.29.0"
+---
+
```

Comment:
I wonder if that should be rather, given `---` is start of the document. The L24 should be rather omitted? 

```
---
schema: olm.channel
name: stable
package: serverless-operator
entries:
  - name: "serverless-operator.v1.27.0"
  - name: "serverless-operator.v1.28.0"
    replaces: "serverless-operator.v1.27.0"
    skipRange: ">=1.27.0 <1.28.0"
  - name: "serverless-operator.v1.29.0"
    replaces: "serverless-operator.v1.28.0"
    skipRange: ">=1.28.0 <1.29.0"
---
schema: olm.channel
name: serverless-1.29
package: serverless-operator
entries:
  - name: "serverless-operator.v1.27.0"
  - name: "serverless-operator.v1.28.0"
    replaces: "serverless-operator.v1.27.0"
    skipRange: ">=1.27.0 <1.28.0"
  - name: "serverless-operator.v1.29.0"
    replaces: "serverless-operator.v1.28.0"
    skipRange: ">=1.28.0 <1.29.0"
```

Reply:
@Kaustubh-pande ^^

Reply:
@dsimansk I'll follow up with the new PR in release-1.29, Required tests are passing don't want to block the cut-off.

Reply:
We can keep iterating on this one, IMO. Probably we need to figure out the right format anyway not to break operator tests at least.